frontend/public/components/utils/button-bar.jsx

```diff
@@ -36,17 +36,21 @@ const SuccessMessage = ({ message }) => (
 // NOTE: DO NOT use <a> elements within a ButtonBar.
 // They don't support the disabled attribute, and therefore
 // can't be disabled during a pending promise/request.
-/** @type {React.SFC<{children: any, className?: string, errorMessage?: string, infoMessage?: string, successMessage?: string, inProgress: boolean}}>} */
+/** @type {React.SFC<{children: any, className?: string, errorMessage?: string, infoMessage?: string, successMessage?: string, inProgress: boolean, dataTestId?: string}}>} */
 export const ButtonBar = ({
   children,
   className,
   errorMessage,
   infoMessage,
   successMessage,
   inProgress,
+  dataTestId,
```

Comment:
Best to keep the prop name consistent everywhere.
```suggestion
  'data-test-id': dataTestId,
```

But also, you don't _need_ this for your PR because the children you pass in can be scoped to the form itself and it can have a `data-test-id`.